doc/test-framework/writing-e2e-tests.md

```diff
@@ -275,6 +275,12 @@ $ kubectl create -f deploy/operator.yaml --namespace operator-test
 $ operator-sdk test local ./test/e2e --namespace operator-test --no-setup
 ```
 
+### Test user setup
+
```

Comment:
I understand that it is not required by default. Am I right? 
It would be a valid setup for a specific scenario. Could you please add when and why it is required/useful?

Reply:
It depends on what your 'default' is, I suppose. If you were developing an operator on an OpenShift  cluster, my understanding is users there are not cluster admin by default, so you would have to do this to get your tests to run. On Minikube (which is what I use), everything is cluster-admin'd by default so you wouldn't ever have to do this.

Reply:
HI @jberkhahn,

I understand. However, note that the projects scaffolded by the tool has not the OCP schemas. So, I am afraid that all examples are not done guessing that the user is playing with OCP cluster and is not admin.  I mean all goes from the basic principals, and my suggestion here just clarifies exactly when and why would be required to do this setup. 

Something such as: 

### Configuration for restrictive clsuters

For the tests be executed they requiring the same operator's permissions. In this way, if you are running it against an OpenShift cluster and you are not logged with the admin user may be required to grant these permissions to your user. 

This permission can be grant by creating a Role to be applied ..

Is it make sense?

Reply:
Sure, I can add a bit like that.